assets/scss/base/typography/_text.scss

```diff
@@ -48,3 +48,20 @@ Styleguide Typography.small
 small {
   @include core-16;
 }
+
+/*
+Bold regular text
+
+Experimental: Bold regular text
+
+Markup:
+<p class="bold-regular">Some text</p>
+
+Styleguide Typography.bold
+*/
+
+.bold-regular {
+  @include bold-19();
```

Comment:
This already exists https://github.com/hmrc/assets-frontend/blob/70942f652d9129cfd818d5a6a7978e95b5967853/assets/scss/base/typography/_headings.scss#L44